recipes/pyside2/bld.bat

```diff
@@ -0,0 +1,56 @@
+
+set CMAKE_CONFIG="Release"
+
+cd %SRC_DIR%\sources\shiboken2
+mkdir build && cd build
+
+cmake -LAH -G"NMake Makefiles"                               ^
```

Comment:
How about using `Ninja` to build parallelly?

Reply:
It has no advantages over make -jX. Besides it has also limitations on how rules depend on each other/explicit rules, I already seen this for openturns.

Reply:
I meant in the windows build, `nmake` builds serially right?